CMakeLists.txt

```diff
@@ -31,11 +31,6 @@ if (CMAKE_CXX_COMPILER_ID MATCHES Clang)
         "${CMAKE_CXX_FLAGS_RELEASE} -Wno-unused-parameter")
 endif()
 
-if ((NOT "${CMAKE_CXX_FLAGS_RELEASE}" MATCHES "std=c\\+\\+0x") AND
-    (NOT "${CMAKE_CXX_FLAGS_RELEASE}" MATCHES "std=c\\+\\+11"))
-    message(FATAL_ERROR "CXX flags not set properly. Remove CMakeCache.txt and run cmake again")
-endif()
-
```

Comment:
I think so, let's make the error message more helpful. Checking for compiler flags in the main CMakeLists.txt is not robust, as some compilers use incompatible flags.